build.gradle

```diff
@@ -212,6 +212,9 @@ task downloadUnixUid(type: Download) {
   src "https://github.com/FAForever/uid/releases/download/v${faf_uid_version}/faf-uid"
   dest file("${buildDir}/resources/native/faf-uid")
   onlyIfNewer true
+  doLast {
+    file("${buildDir}/resources/native/faf-uid").setExecutable(true, false)
```

Comment:
there should already exist some java code doing that when the uid executable is called

Reply:
Well that code does not work then

Reply:
> there should already exist some java code doing that when the uid executable is called

Does not seem to be the case, see https://github.com/FAForever/downlords-faf-client/blob/develop/src/main/java/com/faforever/client/legacy/PosixUidService.java
Would be weird to do this every time that exe is called.